forge-gui/src/main/java/forge/player/HumanPlay.java

```diff
@@ -250,7 +250,9 @@ else if (costPart instanceof CostPartMana && parts.size() < 2) {
                     || part instanceof CostRemoveCounter
                     || part instanceof CostRemoveAnyCounter
                     || part instanceof CostMill
-                    || part instanceof CostSacrifice) {
+                    || part instanceof CostSacrifice
+                    || part instanceof CostCollectEvidence){
```

Comment:
yes, indent is better – just restore spacing and delete line 255 to get the style tidied up

Reply:
So `CostCollectEvidence) {` instead of `CostCollectEvidence){`

Reply:
alright done. At this point this PR might best be squash merged ?

Reply:
Yeah, I'm not even through all the scripts yet 😅 

Sorry if it's a lot of update pushes... but that's the price you pay for 20+ new scripts in one PR 😬